Examples/UIExplorer/js/ListViewExample.js

```diff
@@ -62,9 +62,9 @@ var ListViewSimpleExample = React.createClass({
         noSpacer={true}
         noScroll={true}>
         <ListView
+          removeClippedSubviews={true}
```

Comment:
Didn't know it was enabled by default, thanks! I'll remove it from here then - keep it simple.